x-pack/plugins/canvas/server/plugin.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CoreSetup, PluginInitializerContext } from 'src/core/server';
+import { initRoutes } from './routes';
+
+/**
+ * Represents Code Plugin instance that will be managed by the Kibana plugin system.
+ */
+export class CanvasPlugin {
```

Comment:
You should import and implement the `Plugin` interface from `src/core/server` here.